### DISCUSSION TITLE
I want to assign values to nodes from a script

### DISCUSSION THREAD

**Discussion by Sdphu** [bnpr/Malt#506] [Q&A] [answered]
For example, if I want to assign a value to the input "A" of the Math-Float-Add node, I understand that the value is stored in node_groups["Malt Node Tree"].malt_parameters["_U_0_MALT_NODE_TREE_float1_0_a"].
However, I'm having difficulty assigning values to specific nodes using this method, so I need some help.

**Comment by Sdphu** [comment, marked as answer]
After reading references and trying several methods at random, I finally found one method, which I leave here.
nodes["Float"].inputs["a"].get_source_global_reference()

**Reply by pragma37** [reply]
Sorry for not being able to answer earlier.
The API was not really designed for external use and now I realized that probably was a mistake.
That's indeed how you can retrieve the actual parameter reference, just keep in mind that some property types are stored in their own arrays inside `MaltPropertyGroup`.
